src/panels/lovelace/cards/types.ts

```diff
@@ -329,6 +329,7 @@ export interface PictureCardConfig extends LovelaceCardConfig {
   hold_action?: ActionConfig;
   double_tap_action?: ActionConfig;
   theme?: string;
+  alt_text?: string;
```

Comment:
Need an accompanying PR for documenting this.

Reply:
I believe https://github.com/home-assistant/home-assistant.io/pull/24990 fulfills that (though I need to revert the PictureEntityCard docs). Please let me know if I missed something!

Reply:
Yeah I saw it eventually and reviewed.  My bad.